demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -634,17 +619,17 @@ static uint32_t ulGlobalEntryTimeMs;
  */
 static OtaAppBuffer_t otaBuffer =
 {
-    .pUpdateFilePath    = updateFilePath,
+    .pUpdateFilePath    = ucUpdateFilePath,
     .updateFilePathsize = OTA_MAX_FILE_PATH_SIZE,
-    .pCertFilePath      = certFilePath,
+    .pCertFilePath      = ucCertFilePath,
     .certFilePathSize   = OTA_MAX_FILE_PATH_SIZE,
-    .pDecodeMemory      = decodeMem,
+    .pDecodeMemory      = ucDecodeMem,
     .decodeMemorySize   = otaconfigFILE_BLOCK_SIZE,
-    .pFileBitmap        = bitmap,
+    .pFileBitmap        = ucBitmap,
     .fileBitmapSize     = OTA_MAX_BLOCK_BITMAP_SIZE,
-    .pUrl               = updateUrl,
+    .pUrl               = ucUpdateUrl,
     .urlSize            = OTA_MAX_URL_SIZE,
-    .pAuthScheme        = authScheme,
+    .pAuthScheme        = ucAuthScheme,
     .authSchemeSize     = OTA_MAX_AUTH_SCHEME_SIZE
 };
```

Comment:
File scope variables are supposed to be the last thing before the function definitions. This means that they should also be after function prototypes.

Also, the comment for each of these needs to be in this format:
/* description
more description. */

instead of

/*
 * Description.
 */

Reply:
I believe your comment is for `xOtaTopicFilterCallbacks`. However the callbacks need to be defined before their usage.

Reply:
Variable kept before private functions to be consistent with demos like coreHTTP, defender and jobs.